business_requirement_deliverable_project/views/business_view.xml

```diff
@@ -56,15 +77,68 @@
                                widget="statinfo"/>
                     </button>
                 </xpath>
-
                 <xpath expr="//field[@name='level']"
                        position="after">
                     <field name="linked_project"/>
                 </xpath>
-
             </field>
         </record>
-
+        <record model="ir.ui.view" id="view_business_requirement_kanban_project">
+            <field name="name">business.requirement.kanban</field>
+            <field name="model">business.requirement</field>
+            <field name="arch" type="xml">
+                <kanban default_group_by="state">
+                    <field name="state"/>
+                    <field name="name"/>
+                    <field name="description"/>
+                    <field name="responsible_id"/>
+                    <field name="project_id"/>
+                    <field name="category_ids"/>
+                    <field name="priority"/>
+                    <field name="requested_id"/>
```

Comment:
My last comment was not enough: I meant to remove the request_id and replace by responsible_id.
@sudhir-serpentcs sorry about that